netbox_routing/models/ospf.py

```diff
@@ -54,11 +54,26 @@ def get_absolute_url(self):
 
 class OSPFArea(PrimaryModel):
     area_id = models.CharField(max_length=100, verbose_name='Area ID')
+    area_type = models.CharField(
+        verbose_name=_('Area Type'),
+        help_text=_('OSPF Area Type'),
+        choices=choices.OSPFAreaTypeChoices,
+        default=choices.OSPFAreaTypeChoices.STANDARD,
+        blank=False,
+        null=False
+    )
+    instance = models.ForeignKey(
+        verbose_name=_('OSPF Instance'),
+        help_text=_('OSPF Instance where this area belongs'),
+        to=OSPFInstance,
+        related_name='ospf_areas',
+        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+    )
```

Comment:
Lets not add instance here for now.  This is unrelated to the task at hand (adding OSPF Area Types)